mkdocs.yml

```diff
@@ -45,6 +45,12 @@ nav:
   - Performance Tools:
       - Benchmarking: 'performance-tools/benchmark.md'
       - Benchmarking APIs: 'performance-tools/api-docs.md'
+  - Use Cases: 
+      - Automated Self Checkout: 'use-cases/automated-self-checkout/automated-self-checkout.md'
+      - Getting Started: 'use-cases/automated-self-checkout/getting_started.md'
+      - Advanced Settings: 'use-cases/automated-self-checkout/advanced.md'
+      - Performance: 'use-cases/automated-self-checkout/performance.md'
```

Comment:
should these navigation items become sub-items of the first one: automated self-checkout?

Reply:
Good idea. That will allow us to add additional use cases in the future.